code/form/LocatorForm.php

```diff
@@ -30,11 +44,19 @@ public function __construct(Controller $controller, $name)
         if ($categories) {
             $categoriesField = DropdownField::create('CategoryID')
                 ->setTitle('')
-                ->setEmptyString('All Categories')
+                ->setEmptyString('all categories')
                 ->setSource($categories->map());
             $fields->push($categoriesField);
         }
 
+        if (Config::inst()->get('LocatorForm', 'show_radius')) {
+            $radiusArray = Config::inst()->get('LocatorForm', 'radius_array');
+            debug::show($radiusArray);
```

Comment:
Something like the following would allow the developer to override/alter the radius array.

```php
$this->extend('overrideRadiusArray', $radiusArray);
```